src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -873,6 +901,8 @@
     short-summary: The path to the template file.
   - name: --template-uri -u
     short-summary: The URI to the template file.
+  - name: --template-spec -t
+    short-summary: The template spec resource id.
```

Comment:
`short-summary` in `_help.py` overlapped with `help` declaration in the `deployment_template_spec_type` in `_params.py`, is it possible to consider keeping only declaration in `deployment_template_spec_type`?